Observing-Interface-Setup/efd-kafka.rst

```diff
@@ -0,0 +1,109 @@
+#############
+EFD SAL Kafka 
+#############
+
+Overview
+^^^^^^^^
+This topic covers information on how to perform actions on the EFD SAL Kafka producers/writers on the non-production(non-zero indexed) domains.
```

Comment:
Ok, so you are documenting a development procedure that is not intended for regular users. I think you should make it very clear at the beginning. Try starting with something like:


> In some special occasions a component may be present at the summit but still in early stages of integration and verification. In these conditions it may happen that the responsible software engineer needs to perform rapid changes to the component interface and/or software that are not compatible with the currently running production environment.
>
> In order to avoid conflicts with the running systems, engineers can run their system isolated from the rest of the system by use of a different DDS domain.
>  Still, scientists and engineers may be interested in having the data produced by the component ingested into the EFD for later analysis.
> In order to support this activities it is possible to run alternative kafka producers specially tailored for the running activity.
>
> This procedure details how to provision a SAL Kafka producers on non-production (non-zero indexed) DDS domains to support development activities at the summit.